lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -102,6 +102,14 @@ static CodableConformanceType typeConformsToCodable(DeclContext *context,
 /// \param proto The \c ProtocolDecl to check conformance to.
 static CodableConformanceType
 varConformsToCodable(DeclContext *DC, VarDecl *varDecl, ProtocolDecl *proto) {
+  // CodingKeys enum that couldn't be generated correctly because of some other
+  // error elsewhere. e.g. Invalid redeclaration of members. So let's fail early
+  // here as type not validated.
```

Comment:
Can we drop or at least reduce this comment? The early return is pretty self-explanatory.